app/models/event.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class Event < ApplicationRecord
+  validates :title, :description, :location, :date, presence: true
```

Comment:
Can two events with the same title, description, location and date exist? [OPTIONAL]